### PR TITLE
chore: ask for node type in `bug.yml` issue template

### PR DESCRIPTION
**Pull request by shekhirin**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -67,6 +67,16 @@ body:
...
     validations:
       required: false
+  - type: dropdown
+    id: node
+    attributes:
+      label: What type of node are you running?
+      options:
+        - Archive (default)
+        - Full via --full flag
+        - Pruned with custom reth.toml config
```
Comment (Rjected) [inline .github/ISSUE_TEMPLATE/bug.yml:77]: should ask for what the full config, if custom pruned

**Review by Rjected — APPROVED** [review]
lgtm other than the comment on asking for custom config

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (rkrasiuk) [reply .github/ISSUE_TEMPLATE/bug.yml:77]: that can be added in additional context

**Review by rkrasiuk — APPROVED** [review]
lgtm

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (shekhirin) [reply .github/ISSUE_TEMPLATE/bug.yml:79]: added

**Review by mattsse — APPROVED** [review]

**Pull request merged**